GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -149,6 +149,20 @@ - (void)testUpdateAuthState_tokensAreNotChanged {
   XCTAssertIdentical(user.refreshToken, refreshTokenBeforeUpdate);
 }
 
+- (void)testFetcherAuthorizer {
+  // This is really hard to test without assuming how GTMAppAuthFetcherAuthorization works
+  // internally, so let's just take the shortcut here by asserting we get a
+  // GTMAppAuthFetcherAuthorization object.
+  GIDGoogleUser *user = [self googleUserWithAccessTokenExpiresIn:kAccessTokenExpiresIn
+                                                idTokenExpiresIn:kIDTokenExpiresIn];
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+  id<GTMFetcherAuthorizationProtocol> fetcherAuthroizer = user.fetcherAuthorizer;
+#pragma clang diagnostic pop
+  XCTAssertTrue([fetcherAuthroizer isKindOfClass:[GTMAppAuthFetcherAuthorization class]]);
```

Comment:
We could probably use [`isMemberOfClass:`](https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418766-ismemberofclass) here instead of [`isKindOfClass:`](https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418511-iskindofclass).

Reply:
Yes, you're right!  I forgot that we have our own subclass.